src/Crocodealer/Command/Sync.hs

```diff
@@ -0,0 +1,54 @@
+module Crocodealer.Command.Sync
+       ( sync
+       ) where
+
+import Network.HTTP.Simple (getResponseBody, getResponseStatus, httpBS, parseRequest)
+import Network.HTTP.Types.Status (Status (..))
+
+import Crocodealer.ColorTerminal (errorMessage, infoMessage, warningMessage)
+import Crocodealer.Core.GitHub (FileName (..), RepoName (..), UserName (..))
+
+
+data FileStatus
+    = UpToDate
+    | Outdated
+    | FileDoesNotExist
+
+sync :: UserName -> RepoName -> FileName -> [RepoName] -> IO ()
+sync userName templateRepoName fileName repos =
+    fetchGitHubFile userName templateRepoName fileName >>= \case
+        Nothing -> printFileDiff userName templateRepoName FileDoesNotExist
```

Comment:
I maybe wrong, but is this the only reason the request can fail? I think there could be more stuff going on in the `fetchGitHubFile`. Do you think we can make this error more descriptive?

Reply:
That's true, it could be network issue or something like that. Since we will move to using `curl` here, do you think this is worth investing into?

Reply:
Oh, sorry, I've just noticed this comment. Good point, no need to do it at the moment!